utils/pow.c

```diff
@@ -39,12 +39,17 @@ flex_trit_t* ta_pow_flex(const flex_trit_t* const trits_in, const uint8_t mwm) {
 status_t ta_pow(const bundle_transactions_t* bundle,
                 const flex_trit_t* const trunk, const flex_trit_t* const branch,
                 const uint8_t mwm) {
+  status_t ret = SC_OK;
   iota_transaction_t* tx;
   flex_trit_t* ctrunk =
       (flex_trit_t*)calloc(FLEX_TRIT_SIZE_243, sizeof(flex_trit_t));
   size_t cur_idx = 0;
 
   tx = (iota_transaction_t*)utarray_front(bundle);
+  if (tx == NULL) {
```

Comment:
Why would you change `pow.c`?

Reply:
when I run test driver.c, `pow.c` would cause some memory leak problems

Reply:
Send another pull request to resolve PoW memory leaks.

Reply:
OK